vmm/src/vm.rs

```diff
@@ -1576,6 +1614,16 @@ impl<'a> Vm<'a> {
         // Unblock all CPU threads.
         vcpu_thread_barrier.wait();
 
+        if let Some(console_input) = &self.devices.console_input {
+            let console_input_clone = console_input.clone();
+            let signals = Signals::new(&[SIGWINCH]);
+            match signals {
+                Ok(sig) => {
+                    thread::spawn(move || Vm::handler(sig, console_input_clone));
+                }
+                Err(_) => error!("Signal not found"),
```

Comment:
Maybe use the actual error returned.

Reply:
Printed the error along with this message.